README.md

```diff
@@ -16,34 +16,34 @@ The following configuration steps must be completed before a C1T vehicle or simu
     > [NOTE]
     > See [1. udev Rules Setup](https://f1tenth.readthedocs.io/en/foxy_test/getting_started/firmware/drive_workspace.html#udev-rules-setup) in the F1Tenth build documentation for details.
 
-3. Source the Nav2 install for future terminal instances with `echo 'source /home/$USER/c1t_ws/install/setup.bash' >> ~/.bashrc`
+3. Source the Nav2 install for future terminal instances with `echo 'source /home/$USER/cda_ws/install/setup.bash' >> ~/.bashrc`
 
 4. A map and route graph will need to be created for physical test environments. The `turtlebot` configuration includes these for reference. Maps can be created using [slam_toolbox](https://github.com/SteveMacenski/slam_toolbox) and graphs should follow the examples given in this package. New maps and graphs will need to manually updated in the launch and parameter files.
 
 
 ## Launch
 
-Run the following command to launch the C1T system:
+Run the following command to launch the CDA1Tenth system:
 
 ```
-$ ros2 launch c1t_bringup c1t_bringup_launch.xml vehicle:=[red_truck, blue_truck, turtlebot] record_bag:=[true, false]
+$ ros2 launch cda1tenth_bringup cda1tenth_bringup_launch.xml vehicle:=[red_truck, blue_truck, turtlebot] record_bag:=[true, false]
 ```
 
 This will launch all necessary subsystems, such as drivers, localization, and navigation.
 
 ### Launch Arguments
 - `vehicle` controls the parameter file used to launch the system and requires an input
 
-  - `red_truck` and `blue_truck` target parameter files specific to their respective C1T vehicles
+  - `red_truck` and `blue_truck` target parameter files specific to their respective CDA1Tenth vehicles
 
-  - `turtlebot` runs a Gazebo-based simulation that allows for running core `c1t_bringup` packages locally without hardware interface nodes
+  - `turtlebot` runs a Gazebo-based simulation that allows for running core `cda1tenth_bringup` packages locally without hardware interface nodes
 
-- `record_bag` is a boolean argument for recording all data from the system in a ROS2 rosbag and defaults to `false`. Bags are saved in `~/c1t_bags` along with a copy of the parameter file used
+- `record_bag` is a boolean argument for recording all data from the system in a ROS2 rosbag and defaults to `false`. Bags are saved in `~/cda_bags` along with a copy of the parameter file used
 
 
 ## Post launch
 
-After launching the nodes included in `c1t_bringup`, additional actions are needed to finalize the system on a physical vehicle.
+After launching the nodes included in `cda1tenth_bringup`, additional actions are needed to finalize the system on a physical vehicle.
 
 1. Send an empty message to the `/ackermann_cmd` topic to initialize the VESC by publishing via command line on the vehicle with `ros2 pub /ackermann_cmd ackermann_msgs/msg/AckermannDriveStamped`. Use `CTRL + C` after a few have sent and the original terminal output has updated.
```

Comment:
Should we remove this? Or put it as a footnote for reference

Reply:
I think we can remove this and include it in a debugging document of some sorts